src/components/Dashboard/Carousel/Carousel.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const Carousel = ({ children, id, color }) => {
```

Comment:
Remember we use named exports
```suggestion
export const Carousel = ({ children, id, color }) => {
```